packages/wdio-logger/src/node.ts

```diff
@@ -83,7 +83,7 @@ const wdioLoggerMethodFactory = function (methodName, logLevel, loggerName) {
             return arg
         })
 
-        const logText = ansiStrip(`${util.format.apply(this, args)}\n`)
+        const logText = ansiStrip(`${util.format.apply(this, args as [string])}\n`)
```

Comment:
As discussed before, I think the `as [string]` can go because in reality we pass the entire array